### DISCUSSION TITLE
No examples with arrays in the docs?

### DISCUSSION THREAD

**Discussion by rodrigocfd** [vuejs/pinia#934] [Help and Questions] [answered]
I was looking for an example of how to store an array in a store, and after read through the whole documentation, I found zero examples.

Examples using arrays would greatly help the newcomers.

**Comment by posva** [comment, marked as answer]
You just use an array like any other variable in JS:

```js
defineStore('id', {
  state: () => ({ list: [] })
})
```

**Reply by 0106WeiWeiDeng** [reply]
Hello! In typescript, `defineStore('id', {
  state: () => ({ list: [] })
})`, **list** is `never[]` type, what should I do, if I want to get a `string[]` type **list**. Thanks!

**Reply by posva** [reply]
Do [] as string[]
